nextcord/timestamp.py

```diff
@@ -0,0 +1,72 @@
+from dataclasses import dataclass
+from datetime import datetime as dt
+
+
+@dataclass
+class Timestamp:
+    """A class to represent Discord's native timestamp format."""
+
+    time: int
```

Comment:
I think we should pass in a `datetime` object over a timestamp.

Very rarely am I dealing with a unix timestamp directly; I've almost always had a `datetime` that I'd like to display directly.